### ISSUE TITLE
Pantheon still can't count!!

### ISSUE THREAD

**Issue by AdrianPKnowles** [BiologicalRecordsCentre/Pantheon#258]
I'm getting rather frustrated that Pantheon still has fundamental errors, like the inability to count.  For a species list I have just entered the summary table says I have 18 Hemiptera.  Clicking on the hyperlink for "18" is displays a list of 17 species.....

One wonders if the problem is Nysius huttoni, which appears in the "not analysed" list but is possibly being counted twice for some reason?

**Comment by Davydaveth** [comment]
No-one is really running on here as we could get no project funds. You are best running this sort of point via email at the moment.
That said, does the taxonomic output page at the start of the analysis give the correct number of bugs?
David

**Comment by AdrianPKnowles** [comment]
David

Thanks for your reply.  The total number of Hemiptera in my list is 18.  The “Results overview” table says 18 for the number analysed, but there are 17 species in that list when you click on the hyperlink, because the 18th species, Nysius huttoni, appears in the “not analysed” list.  I think I’ve flagged up issues with N. huttoni before.

 

It is a pity that funding has dried up because I fear there are still several flaws in the data output that would lead the unwary user astray e.g. there were dozens of S41 Priority species that did not have that conservation value listed in the species list (although I think the correct SQI value may be being used in the SQI calculation).

 

Best wishes

 

Adrian

**Comment by Davydaveth** [comment]
I’m going to make a play for underspend this autumn, as without some decent funding it is hard to John to commit too much time to fix issues, let along increase analytical capacity. Useful to know how it differentially counts. Perhaps a note on that point might be useful as the distinction between analysed and non-analysed samples data is important.

We have an editing portal that can make, although not live, write changes to the database, so if you come across things you find send them over. We still have to wait for a database refresh for them to show though.
Be safe
David

Senior Entomologist
Natural England
Chief Scientist Directorate:
Specialist Services & Programmes
Natural England
County Hall,
Spetchley Road,
Worcester WR5 2NP

Jabber Internal number 60947 (Worcester)
Jabber external number 02080260947
